core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -34,16 +40,17 @@ export interface QuestionBackendDict {
   'inapplicable_skill_misconception_ids': string[];
 }
 
+// Null will be temperory and will be updated once the question is saved.
```

Comment:
```suggestion
// Null in ID is only temporary and will be updated once the question is saved.
```

Reply:
Is this comment a duplicate of the comment that is already below? 
```
export class Question {
  // A null '_id' indicates that the 'Question' has been created
  // but not saved.
  _id: string | null;
  ...
```

Reply:
Yes, it's a duplicate, will remove it. Thanks!